packages/ocplib_stuff/ocplib_stuff.0.3.0/opam

```diff
@@ -38,7 +38,7 @@ depends: [
   "ppx_expect" {with-test}
   "odoc" {with-doc}
   "ocamlformat" {with-test}
-  "dune" {>= "2.6.0" & < "3.0.0"}
+  "dune" {>= "2.6.0"}
```

Comment:
There appear to be duplicated dependencies on `dune` here.

Reply:
Yes, `drom` does not try to simplify dependencies when it gathers them from different sources, but it should not be a problem for `opam` to manage that correctly.

Reply:
```suggestion
```